CHANGELOG.md

```diff
@@ -33,216 +44,6 @@ All notable changes to this project will be documented in this file. The format
 
 - MODULES-10956 remove redundant code in provider apt\_key [\#973](https://github.com/puppetlabs/puppetlabs-apt/pull/973) ([moritz-makandra](https://github.com/moritz-makandra))
 
-## [v8.0.0](https://github.com/puppetlabs/puppetlabs-apt/tree/v8.0.0) (2021-03-01)
```

Comment:
Unsure why this is happening.
It hasn't in any of the other modules we've put the issues pin in